wscript

```diff
@@ -317,6 +319,11 @@ def build(bld):
 
     bld.load('ardupilotwaf')
 
+    bld.env.AP_LIBRARIES_OBJECTS_KW.update(
+        use='mavlink',
```

Comment:
If we are forcing the mavlink use here we should remove it from the calls to ap_stlib right?